samples/OpenIdConnect.AzureAdSample/Readme.md

```diff
@@ -0,0 +1,30 @@
+# How to set up the sample
+
+## Pre-requisit
+1. Set up an Azure Active Directory, save the AD's "Tenent name"
+2. Set up an Azure Web Site
```

Comment:
Were you able to run the sample locally? You should be able to if you set the right UserSecrets.